internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -30,8 +30,8 @@
   PortMappings:
     - ContainerPort: !Ref ContainerPort
     {{- if .ServiceConnect }} # remove when release
-    {{- if eq .HTTPTargetContainer.Container .WorkloadName}}
-      Name: TargetPort
+    {{- if eq .HTTPTargetContainer.Name .WorkloadName}}
+      Name: target
```

Comment:
could we combine these conditions with `and` like below

Reply:
it's fine. we'll get rid of the sc check condition when we release this feature.